src/Hit/Git.hs

```diff
@@ -73,6 +73,10 @@ runNew issueOrName = do
                        || isSpace c
                        || c `elem` ("_-./" :: String)
                    )
+runNew _ title = do
+  createIssue' title >>= \case
+    Left err -> errorMessage $ show err
+    Right issue -> putTextLn . showIssueName blueCode $ issue
```

Comment:
Got the following error (422) when trying to create new issue. (I can successfully create new issue using `hub` tho)
```
~/…/haskell/kowainik/hit-off$ export GITHUB_TOKEN=$MY_GITHUB_TOKEN
~/…/haskell/kowainik/hit-off$ ./hit new -i "Test hit new --issue command"
  HTTPError (HttpExceptionRequest Request {
  host                 = "api.github.com"
  port                 = 443
  secure               = True
  requestHeaders       = [("Authorization","<REDACTED>"),("User-Agent","github.hs/0.7.4"),("Accept","application/vnd.github.preview")]
  path                 = "/repos/kowainik/hit-off/issues"
  queryString          = ""
  method               = "POST"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}
 (StatusCodeException (Response {responseStatus = Status {statusCode = 422, statusMessage = "Unprocessable Entity"}, responseVersion = HTTP/1.1, responseHeaders = [("Date","Fri, 04 Oct 2019 13:01:01 GMT"),("Conte
nt-Type","application/json; charset=utf-8"),("Content-Length","157"),("Server","GitHub.com"),("Status","422 Unprocessable Entity"),("X-RateLimit-Limit","5000"),("X-RateLimit-Remaining","4999"),("X-RateLimit-Reset
","1570197661"),("X-OAuth-Scopes","admin:gpg_key, admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user, write:discussion"),("X-Accepted-OAuth-Scopes",""),("X-
GitHub-Media-Type","github.v3; param=preview"),("Access-Control-Expose-Headers","ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Acc
epted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type"),("Access-Control-Allow-Origin","*"),("Strict-Transport-Security","max-age=31536000; includeSubdomains; preload"),("X-Frame-Options","deny"),("X-Content-T
ype-Options","nosniff"),("X-XSS-Protection","1; mode=block"),("Referrer-Policy","origin-when-cross-origin, strict-origin-when-cross-origin"),("Content-Security-Policy","default-src 'none'"),("X-GitHub-Request-Id"
,"C978:1522:72B92A:8507DF:5D97428D")], responseBody = (), responseCookieJar = CJ {expose = []}, responseClose' = ResponseClose}) "{\"message\":\"Invalid request.\\n\\nFor 'properties/labels', nil is not an array.
\",\"documentation_url\":\"https://developer.github.com/v3/issues/#create-an-issue\"}"))
```

Reply:
Will fix by #119